etna/packages/etna-js/utils/fetch.js

```diff
@@ -57,6 +57,17 @@ export const headers = (...types) => {
   return _headers;
```

Comment:
Is this file change really part of this PR?

Reply:
no-no, sorry I accidentally rebased one of view editor commits on it. I tried to recover but seems easier if I merge view editor to master and them merge master into this branch